app/models/cake.rb

```diff
@@ -0,0 +1,9 @@
+class Cake
+  def sweet?
+    true
+  end
+
+  def for_children?
+    true
+  end
```

Comment:
⚠️ **[undercover]** <Untested instance method> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Instance method `for_children?` is missing coverage for line 7 (node coverage: 0.0)
<!-- __reviewdog__:Cg82ZGUyNmEwMTM4Mjg4OTUSCnVuZGVyY292ZXI= -->